pkgs/development/coq-modules/mathcomp-infotheo/default.nix

```
@@ -7,8 +7,10 @@ mkCoqDerivation {
   inherit version;
   defaultVersion = with lib.versions; lib.switch [ coq.version mathcomp-analysis.version] [
     { cases = [ (range "8.15" "8.16") (range "0.5.4" "0.6.2") ]; out = "0.5.1"; }
+    { cases = [ (isGe "8.17") (isGe "0.6.0") ];                  out = "0.5.2"; }
```

```suggestion
    { cases = [ (isGe "8.17") (isGe "0.6.0") ];                  out = "0.5.2"; }
    { cases = [ (range "8.15" "8.16") (range "0.5.4" "0.6.2") ]; out = "0.5.1"; }
```
here it's equivalent, all cases being exclusive, but since it's a match we usually write the most recent version on top